install.js

```diff
@@ -39,4 +39,7 @@ async function load(modulePath) {
   // Install Pandoc binary
   const pandoc = await load('./dist/codecs/pandoc/binary')
   await pandoc.install()
+  // Register Encoda as an executor
+  const encoda = await load('./dist/encoda')
+  encoda.run('register')
```

Comment:
I'm probably missing context here, but I'm not sure if we should be running this as a post-install step.
I believe it'll be triggered when installed as a dependency (e.g. by schema), and I'm not sure what happens if it's registered multiple times.
Overall I think I'd rather see it as an explicit step that gets called upon desire to use it.

Reply:
Yes, I think you are right. I was tempted by avoiding the extra step.